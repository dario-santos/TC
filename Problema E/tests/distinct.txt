4
S -> C
A -> B
B -> S
C -> A
