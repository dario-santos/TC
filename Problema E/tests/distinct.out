NULL(S) = False
NULL(A) = False
NULL(B) = False
NULL(C) = False
FIRST(S) =
FIRST(A) =
FIRST(B) =
FIRST(C) =
FOLLOW(S) = #
FOLLOW(A) = #
FOLLOW(B) = #
FOLLOW(C) = #
